### PR TITLE
[14.0][MIG] account_move_fiscal_month

### PR DESCRIPTION
**Pull request by xavier-bouquiaux**
tests ok <br/>
pre-commit ok <br/>
depends : https://github.com/OCA/account-financial-tools/pull/1242

### CODE REVIEW

**Comment by kittiu** [comment]
Please rebase, the `account_fiscal_month` is already merged.

Note: do you know why build is not failed, while there is no `account_fiscal_month`

**Comment by xavier-bouquiaux** [comment]
@Cedric-Pigeon , @rousseldenis 
-> may-be we can merge this <br/>

**Comment by rousseldenis** [comment]
@kittiu Could you review this ?

File: account_move_fiscal_month/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+* Benjamin Willig <benjamin.willig@acsone.eu>
```
Comment (rousseldenis) [inline account_move_fiscal_month/readme/CONTRIBUTORS.rst:1]: @xavier-bouquiaux Please add yourself

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by AaronHForgeFlow — APPROVED** [review]
Funcional + code review

**Review by JuanyDForgeflow — APPROVED** [review]

**Pull request merged**